sdks/fabric/interoperation-node-sdk/src/EventsManager.ts

```diff
@@ -65,12 +65,19 @@ function createEventPublicationSpec ({
     if (appUrl) {
         eventPublicationSpec.setAppUrl(appUrl)
     } else {
+
+        let ccArgsB64 = [];
```

Comment:
Can you rename this variable to `ccArgsBytes` as we are creating bytes now instead of B64.

Reply:
Done